app/constants.py

```diff
@@ -127,6 +127,12 @@
 NOTIFICATION_TYPES = [SMS_TYPE, EMAIL_TYPE, LETTER_TYPE]  # not broadcast
 NOTIFICATION_TYPE = [EMAIL_TYPE, SMS_TYPE, LETTER_TYPE]  # duplicate that can probably be cleaned up
 
+# Language options supported for bilingual letter templates
+ENGLISH_LANGUAGE = "english"
+WELSH_THEN_ENGLISH_LANGUAGE = "welsh_then_english"
+
+TEMPLATE_LANGUAGE_OPTIONS = [ENGLISH_LANGUAGE, WELSH_THEN_ENGLISH_LANGUAGE]
```

Comment:
question/optional suggestion: What do you think about using an actual enum? See `constants.OrganisationUserPermissionTypes` for an example. Although python enums have a bit of extra faff (having to use `eg LanguageOptions.ENGLISH.value` to get the actual string, it may be a bit clearer in terms of intent, and the intellisense support can be more helpful.

Reply:
I don't use intellisense much, but if it makes life easier for others, it's worth considering. I like the class syntax in it. `LanguageOptions.ENGLISH.value` does sound a bit faffy, I wonder why they implemented it like that 😅 .

Most of all I think having both types of syntax at once is non-consistent. So it would be good to discuss this on the tech catch up today, and decide which way we want to go, going forward.